Add Paradedb blog draft (#668)
tembo-io/website: src/content/blog/2024-10-03-paradedb-search-analytics/index.mdx

---
slug: paradedb-search-analytics
title: 'Announcing ParadeDB partnership: Search and Analytics for Postgres'
authors: [adam]
description: |
We’re launching Postgres for Search and Analytics, as well as efficient analytics on cloud storage, powered by ParadeDB.
tags: [postgresql, analytics, search, paradedb]
date: 2024-10-03T09:00
image: './paradedb_partner.png'
planetPostgres: true
---

We're excited to announce the launch of two new Tembo stacks, powered by ParadeDB.

- The ParadeDB Stack — Designed as a search stack, it is powered by the ParadeDB extensions and is designed as an Elasticsearch alternative for customers needing high-quality full-text search over Postgres data.

> - The Analytics Stack — Designed as a data lake stack, it is powered by the ParadeDB pg_analytics DuckDB-in-Postgres extension and is designed for high-performance analytics over object storages like S3 and table formats like Iceberg.

Reviewer: ```suggestion
- The Analytics Stack — Gives developers a data lake experience on Postgres. It is powered by the ParadeDB pg_analytics extension that brings DuckDB into Postgres and is designed for high-performance analytics over object storages like S3 and table formats like Iceberg.
```

> ## What is ParadeDB?

Reviewer: ```suggestion
## ParadeDB is Search and Analytics for Postgres 
```

[ParadeDB](https://www.paradedb.com/) is an Elasticsearch alternative built on Postgres. ParadeDB is built as two extensions, pg_search and pg_analytics, which together bring fast full-text search and high-performance analytics to Postgres.

> ## Why we're partnering?

Reviewer: ```suggestion
## Run Every Workload on Tembo Postgres
```
or maybe just "Run Every Workload on Postgres"
